sub/sd_sbr.c

```diff
@@ -95,6 +95,21 @@ static int init(struct sd *sd)
     if (strcmp(sd->codec->codec, "subrandr/srv3"))
         return -1;
 
+    uint32_t major, minor, patch;
+    sbr_library_version(&major, &minor, &patch);
+
+    MP_VERBOSE(sd, "subrandr version: %d.%d.%d", SUBRANDR_MAJOR, SUBRANDR_MINOR, SUBRANDR_PATCH);
+    if(SUBRANDR_MAJOR != major || SUBRANDR_MINOR != minor || SUBRANDR_PATCH != patch)
+        MP_VERBOSE(sd, " (runtime %d.%d.%d)", major, minor, patch);
+    MP_VERBOSE(sd, "\n");
+
+    if (major != SUBRANDR_MAJOR || minor < SUBRANDR_MINOR) {
```

Comment:
@afishhh: Is this good assumption of ABI compatibility for subrandr?

Reply:
Yeah it's fine, will work.